paddle/operators/momentum_op.h

```diff
@@ -44,11 +47,28 @@ class MomentumOpKernel : public framework::OpKernel<T> {
     auto g = framework::EigenVector<T>::Flatten(*grad);
     auto* lr = learning_rate->data<T>();
 
+    T local_lr = lr[0];
+    if (use_local_lr) {
```

Comment:
I didn't see the corresponding CUDA implement for this feature. So it's more proper if you can implement this as a "CPU_ONLY_KERNEL" or add CUDA support in both sgd and momentum op.

Reply:
will implement it as a CPU_ONLY_KERNEL firstly. Thanks.